src/sections/footer.html

```diff
@@ -52,20 +49,25 @@ <h1 class="lg:w-80 font-bold text-2xl">
                         </p>
                     </div>
                 </div>
-                
-                <div class="md:py-12 py-4 w-full md:w-3/4 mx-auto md:grid md:grid-cols-2 justify-between gap-9 font-para" id="accordion4">
+
+                <div class="md:py-16 px-4 py-4 w-full md:w-3/4 mx-auto md:grid md:grid-cols-2 justify-between gap-16 font-para"
+                    id="accordion4">
                 </div>
-                
+
                 <div class="py-3">
                     <div class="lg:w-1/2">
-                        <h1 class="font-bold md:text-2xl text-xl py-3">Nehmen Sie Kontakt zu uns auf!
+                        <h1 class="font-bold md:text-2xl text-xl py-3 md:py-6">Nehmen Sie Kontakt zu uns auf!
                         </h1>
                         <button type="button"
-                            class="bg-yellow-500 hover:bg-yellow-600 py-2 px-6 rounded-3xl shadow-sm font-semibold border-none flex gap-3 items-center"> <span><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" class="w-6 h-6">
-                                <path d="M1.5 8.67v8.58a3 3 0 003 3h15a3 3 0 003-3V8.67l-8.928 5.493a3 3 0 01-3.144 0L1.5 8.67z" />
-                                <path d="M22.5 6.908V6.75a3 3 0 00-3-3h-15a3 3 0 00-3 3v.158l9.714 5.978a1.5 1.5 0 001.572 0L22.5 6.908z" />
-                              </svg>
-                              </span> Nachricht schreiben</button>
+                            class="text-backgroundColor bg-yellow-500 hover:bg-yellow-600 py-2 px-6 rounded-3xl shadow-sm font-semibold border-none flex gap-2 md:gap-6 items-center">
```

Comment:
Change the background color of the mail button

Reply:
done